packages/angular/src/builders/utilities/wait-for-port.ts

```diff
@@ -0,0 +1,40 @@
+import * as net from 'net';
+
+export function waitForPortOpen(
```

Comment:
Should we add `@nx/web` as a dep of `@nx/angular`? We're already installed it for module federation for `@nx/web:file-server`. It's a light package.